cpp/src/structs/utilities.cpp

```diff
@@ -24,7 +24,12 @@
 #include <cudf/unary.hpp>
 #include <cudf/utilities/error.hpp>
 #include <cudf/utilities/span.hpp>
+#include <iterator>
 #include <structs/utilities.hpp>
+#include "cudf/types.hpp"
+#include "cudf/utilities/traits.hpp"
+
+#include <bitset>
```

Comment:
Wrong including order and format :)

Reply:
Thanks for pointing that out. I should've checked this before raising the PR. I've to figure out how to instruct the IDE on the right way to include.